.ci/no-exception-test.sh

```diff
@@ -53,6 +53,28 @@ guava-with-sun-checks)
   rm sun_checks.*
   ;;
 
+openjdk14-with-checks-nonjavadoc-error)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: $CS_POM_VERSION
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/contribution
+  cd ..
+  mvn -e clean install -Pno-validations
```

Comment:
please move this to build line to be done before clone of contribution

Reply:
Done.

Reply:
Just curious, why? In case of build failure, we don't waste resources/time to clone the contribution repo?

Reply:
build failure is rare
in CI we do not make waste, as whole environment is deleted after al executions, only logs are left.
on local, yes it might, but usually this command executed the last verification, so build is passing for sure.